ui/src/components/easter/EasterBanner.js

```diff
@@ -1,71 +1,142 @@
-import {
-  Flex,
-  Center,
-  Button,
-  Link,
-  Box
-} from '@chakra-ui/react';
-import { useHistory } from "react-router-dom";
+import { Flex, Center, Button, Link, Box, HStack } from '@chakra-ui/react';
+import { useHistory } from 'react-router-dom';
 
+let IsSite = (array) => {
+  const Site = useHistory().location.pathname.includes(array.site);
+  return Site;
+};
 
 const EasterBanner = () => {
-  const isEasterSite = useHistory().location.pathname.includes("easter");
+  let currDate = new Date().toDateString().substr(0, 10);
```

Comment:
not very minor, it's fine if you don't have time to get to this, but can you look into luxon? we use this date time library for dates in our code, gives us lots of flexibility with these date times

reference: https://github.com/hmcc-global/hmcchk-web/pull/332/files#diff-f0d3326b07b0d77fe379774b5f5c5b1a5a33fef4b6aaf470ec6397e48467b7ddR36